runtime/permissions/lib.rs

```diff
@@ -596,10 +587,26 @@ pub trait QueryDescriptor: Debug {
 }
 
 fn format_display_name(display_name: Cow<str>) -> Cow<str> {
+  fn truncate_to_n(s: &mut String, n: usize) -> bool {
+    for (char_count, (idx, _)) in s.char_indices().enumerate() {
+      if char_count == n {
+        s.truncate(idx);
+        return true;
+      }
+    }
+    false
+  }
+
   if display_name.starts_with('<') && display_name.ends_with('>') {
     display_name
   } else {
-    Cow::Owned(format!("\"{}\"", display_name))
+    let mut display = display_name.into_owned();
+    let truncated = if truncate_to_n(&mut display, 1024) {
+      "...(truncated)"
+    } else {
+      ""
+    };
+    Cow::Owned(format!("\"{}{}\"", display, truncated))
   }
 }
```

Comment:
Is this truncating desirable? I pushed it down from ext/fs. I think probably it's ok because 1024 chars is a very long path?

Reply:
It looks like we have a way of preventing this already, but it's a 10kb max: https://github.com/denoland/deno/commit/00970daea2245bf4af6b3ee21d0e522fec5638b8#diff-2f10844da1901d00c093d0bc0393da74b6deae290c4c26c18d8058b62268575e

I'm just going to remove this code. Seems we didn't use it for most apis before anyway.